beacon_node/eth1/src/service.rs

```diff
@@ -215,16 +217,22 @@ async fn get_remote_head_and_new_block_ranges(
     if remote_head_block.timestamp + node_far_behind_seconds < now {
         warn!(
             service.log,
-            "Eth1 endpoint is far behind. Trying fallback ...";
+            "Eth1 endpoint is not synced";
             "endpoint" => endpoint,
+            "action" => "trying fallback";
```

Comment:
```suggestion
            "action" => "trying fallback",
```

Reply:
damn, I didn't even check it compiles before pushing! thanks :)